src/main/java/eu/dissco/backend/repository/SpecimenRepository.java

```diff
@@ -22,23 +22,18 @@ public class SpecimenRepository {
   private final ObjectMapper mapper;
 
   public List<DigitalSpecimen> getSpecimensLatest(int pageNumber, int pageSize) {
-    var offset = 0;
-    if (pageNumber > 1) {
-      offset = offset + (pageSize * (pageNumber - 1));
-    }
+    int offset = getOffset(pageNumber, pageSize);
     return context.select(NEW_DIGITAL_SPECIMEN.asterisk())
         .from(NEW_DIGITAL_SPECIMEN)
         .offset(offset)
         .limit(pageSize)
         .fetch(this::mapper);
   }
 
-  public DigitalSpecimen getLatestSpecimenById(String id) {
+  public DigitalSpecimen getSpecimenById(String id) {
     return context.select(NEW_DIGITAL_SPECIMEN.asterisk())
```

Comment:
You're still getting the latest specimen by Id, I'd keep the original name (is consistent with annotation repository too)

Reply:
Yes we need to look at it at some point as the same method is also in the ElastiRepository, there is some duplication there. Will revert this here